engine/swarm/how-swarm-mode-works/pki.md

```diff
@@ -53,38 +52,46 @@ Certificate:
-CLI reference.
+You can configure this interval by running the `docker swarm update
+--cert-expiry <TIME PERIOD>` command. The minimum rotation value is 1 hour.
+Refer to the
+[docker swarm update](/engine/reference/commandline/swarm_update.md) CLI
+reference for details.
 
 ## Rotating the CA certificate
 
-In the event that a cluster CA key or a manager node is compromised, the swarm root
-CA can be rotated so that none of the nodes will trust certificates signed by the old
-root CA anymore.
-
-You can run `docker swarm ca --rotate` to tell swarm to generate a new CA certificate
-and key.  If you prefer, you can pass the `--ca-cert` and `--external-ca` flags to specify
-the root certificate and a root CA external to the swarm.  Alternately, you can pass the
-`--ca-cert` and `--ca-key` flags to specify the exact certificate and key you would like
-the swarm to use.
-
-When the command is issued, swarm will generate a cross-signed certificate (a version of the
-new root CA certificate that is signed with the old root CA certificate), and use this
-as an intermediate certificate for all new node certificates.  This is so nodes that still
-trust the old root CA will be able to validate a certificate that is signed by the
-new root CA.
-
-During a root rotation, swarm will also tell all nodes (>= 17.06) to immediately renew
-TLS certificates. Once every node in the swarm has a TLS certificate signed by the new CA,
-(which may take a little while, depending on the number of nodes) swarm will forget about
-the old CA certificate and key material and tell all the nodes to trust the new CA
-certificate only.  This will also cause a change in the swarm's join tokens - the previous
-join tokens will no longer be valid.
-
-From this point on, all new node certificates issued will be signed with the new root CA,
-and will not contain any intermediates.
+In the event that a cluster CA key or a manager node is compromised, you can
+rotate the swarm root CA so that none of the nodes will trust certificates
+signed by the old root CA anymore.
+
+Run `docker swarm ca --rotate` to generate a new CA certificate and key. If you
+prefer, you can pass the `--ca-cert` and `--external-ca` flags to specify the
+root certificate and a use root CA external to the swarm. Alternately, you can
+pass the `--ca-cert` and `--ca-key` flags to specify the exact certificate and
+key you would like the swarm to use.
+
+When you issue the `docker swarm ca --rotate` command, the following things
+happen in sequence:
+
+1.  Docker generates a cross-signed certificate. This means that a version of
+    the new root CA certificate is signed with the old root CA certificate.
+    This cross-signed certificate is used as an intermediate certificate for all
+    new node certificates. This ensures that nodes that still trust the old root
+    CA will be able to validate a certificate signed by tne new CA.
+
+2.  In Docker 17.06 and higher, Docker also tells all nodes to immediately
```

Comment:
Non-blocking:  It's true that only a 17.06 manager will tell nodes to do this, but I think only 17.06 nodes will actually respect the message - should we mention that, in case the user has a mixed cluster?